Serverless Changelog for December 16, 2025 (#4357)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

* Stores the `@timestamp` field value range in the compound commit header
* Uses the existing `DocumentMapper` when creating a new `MapperService` [#138489]({{es-pull}}138489)
* Uses the new bulk scoring dot product for max inner product [#139409]({{es-pull}}139409)

### Fixes [serverless-changelog-12162025-fixes]

* Enables storing secrets in {{fleet}} Server Host config if {{fleet-server}} is running at a minimum supported version [#237464]({{kib-pull}}237464)
* Fixes Discover tab initialization [#245752]({{kib-pull}}245752)
* Improves error handling for tool responses [#241425]({{kib-pull}}241425)
* Updates Gemini connector configuration [#245647]({{kib-pull}}245647)
* Limits the API for retrieving gap summaries to 100 `rule_id`s per request [#245924]({{kib-pull}}245924)
* Fixes "now" and mixed-format date handling in the share modal [#245539]({{kib-pull}}245539)
* Ensures chart tooltips are always shown correctly in anomaly detection result views [#246077]({{kib-pull}}246077)
* Turns off geopoint mapping in the processing preview [#245506]({{kib-pull}}245506)
* Validates child stream input [#242581]({{kib-pull}}242581)

> * Fixes an issue where the upgrade assistant would incorrectly warn about nodes breaching the low watermark despite the max headroom setting [#243906]({{kib-pull}}243906)

Reviewer: ```suggestion
* Fixes an issue where the upgrade assistant would incorrectly warn about nodes breaching the low watermark despite the `max_headroom` setting [#243906]({{kib-pull}}243906)
```

* Fixes an ECS-incompatible value in the logs [#245706]({{kib-pull}}245706)
* Fixes grammatical issues in the Solution Nav tour and simplifies the content by consolidating multiple links into one [#245718]({{kib-pull}}245718)
* Fixes Discover trace waterfall behavior with duplicate spans [#244984]({{kib-pull}}244984)
* Avoids JVM metric conflicts with explicit cast [#244151]({{kib-pull}}244151)
* Fixes an issue where metadata filtering was confusing or broken when typing "OR" in Host view [#233836]({{kib-pull}}233836)
* Compares {{esql}} query builders using identity [#139080]({{es-pull}}139080)
* Adds support for chunking settings for sparse embeddings in a custom service to the Inference API [#138776]({{es-pull}}138776)

> * Uses the `dimensions` field in JinaAI text_embedding requests to the Inference API [#139395]({{es-pull}}139395)

Reviewer: ```suggestion
* Uses the `dimensions` field in JinaAI `text_embedding` requests to the Inference API [#139395]({{es-pull}}139395)
```

* Adds an integration knowledge opt-out UI setting and feature flag [#245080]({{kib-pull}}245080)
* Redesigns the single and bulk agent actions menus in {{fleet}}, organizing commonly used actions at the top level and grouping other actions into nested menus by use case [#245174]({{kib-pull}}245174)
* Adds agent internal YAML settings [#245819]({{kib-pull}}245819)
* Adds support for chain controls [#242909]({{kib-pull}}242909)
* Improves validation and autocomplete for CASE [#244280]({{kib-pull}}244280)
* Avoids redundant requests when breakdown or chart interval changes [#245523]({{kib-pull}}245523)
* Adds support for filtering on multivalue fields [#245554]({{kib-pull}}245554)
* Adds example plugin tests [#245792]({{kib-pull}}245792)
* Adds a **Copy as Markdown** option for selected results [#245545]({{kib-pull}}245545)
* Adds global params sync internal API [#239284]({{kib-pull}}239284)
* Adds the ability to bulk mute and unmute alerts [#245690]({{kib-pull}}245690)

> * Adds Rules feature privileges, allowing access to Rules to be explicitly set for user roles [#239634]({{kib-pull}}239634)

Reviewer 2: ```suggestion
* Adds Rules feature privileges, allowing access to {{elastic-sec}} rules to be explicitly set for user roles [#239634]({{kib-pull}}239634)
```

* Updates the threat hunting UI [#243311]({{kib-pull}}243311)

> * Adds support for QRadar Reference Sets as lookups [#244924]({{kib-pull}}244924)

Reviewer: ```suggestion
* Adds support for QRadar reference sets as lookups [#244924]({{kib-pull}}244924)
```

* Adds an Agent Builder opt-in flow [#244532]({{kib-pull}}244532)
* Redesigns Lookup join file upload [#244550]({{kib-pull}}244550)
* Adds an action to create an anomaly detection alerting rule [#241274]({{kib-pull}}241274)
* Adds an empty state for the Partitioning tab [#244893]({{kib-pull}}244893)
* Improves attachment filters with multi-type selection, server-side filtering, and a suggestions limit [#245248]({{kib-pull}}245248)
* Adds a gap auto-fill scheduler UI and API [#244719]({{kib-pull}}244719)
* Adds a new **Similar errors** section with an occurrences chart [#244665]({{kib-pull}}244665)
* Adds dashboard ownership and write-restricted mode, allowing you to control who can edit your dashboards regardless of broader space permissions [#224552]({{kib-pull}}224552)
* Adds a new gap fill status for rules [#242595]({{kib-pull}}242595)
* Validates space ownership when unlinking attachments [#245250]({{kib-pull}}245250)
* Adds `deactivate_all_instrumentations`, `deactivate_instrumentations`, `send_logs`, `send_metrics`, and `send_traces` agent configuration settings for EDOT PHP [#246021]({{kib-pull}}246021)

> * Adds dashboard suggestions for ECS K8s and OTel Dashboards when selecting pods in the Infra Inventory UI [#245784]({{kib-pull}}245784)

Reviewer: ```suggestion
* Adds dashboard suggestions for ECS Kubernetes and OTel dashboards when selecting pods in the Infrastructure inventory UI [#245784]({{kib-pull}}245784)
```

* Expands search in the All cases page [#245321]({{kib-pull}}245321)
* Adds concurrency to KMeansLocal [#139239]({{es-pull}}139239)
* Enables CCS tests for {{esql}} subqueries [#137776]({{es-pull}}137776)
* Adds CCS support for the {{esql}} Inference command [#139244]({{es-pull}}139244)
* Introduces usage limits for COMPLETION and RERANK [#139074]({{es-pull}}139074)
* Adds privileges to the {{kib}} System role to manage internal indexes in support of {{elastic-defend}} features [#138993]({{es-pull}}138993)
* Optimizes native bulk dot product scoring for Int7 [#139069]({{es-pull}}139069)
* Adds Azure OpenAI chat completion support [#138726]({{es-pull}}138726)

> * Adds NVIDIA support to the Inference Plugin [#132388]({{es-pull}}132388)

Reviewer: Maybe this could be lowercase - inference plugin?
